commons/zenoh-buffers/src/lib.rs

```diff
@@ -102,10 +102,17 @@ pub mod buffer {
             match slices.len() {
                 0 => Cow::Borrowed(b""),
                 1 => Cow::Borrowed(slices.next().unwrap()),
```

Comment:
```suggestion
                1 => {
                    // SAFETY: unwrap here is safe because we have explicitly checked
                    //         the iterator has 1 element.
                    Cow::Borrowed(unsafe { slices.next().unwrap_unchecked() })
                }
```